extensions/extensions-helm/aissemble-airflow-chart/README.md

```diff
@@ -66,8 +65,6 @@ recommend removing them from your values file entirely.
 
 | Old Property Location | New Property Location | Same Default Value | Additional Notes                                                                                                                                                                     |
 |-----------------------|-----------------------|--------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| image.name | defaultAirflowRepository | No                 | This attribute should be combined with `image.dockerRepo`                                                                                                                            |
-| image.dockerRepo| defaultAirflowRepository | No                 | This attribute should be combined with `image.name`                                                                                                                                  |
```

Comment:
S: The migration instructions should remain untouched. It's still accurate to say the old v1 properties map to these properties in the new chart. Unless, that is, the airflow chart had these properties completely removed in the chart upgrade. But if that is the case, I'd expect us to put the new property names here, not remove them entirely.